apiary.apib

```diff
@@ -3250,6 +3250,357 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
```

Comment:
After reading a bit more about work orders I feel like this can also be `.schedule`. Eg. you schedule a work order, and complete it with details of the work you actually performed.

Reply:
After short discussion, it seems that draft is ok.